README.md

```diff
@@ -74,45 +103,53 @@ Total Flash memory (text + data + misc): 60554 bytes
 Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 ```
 
-It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
+After the build completes, the build system creates two binary images in the `BUILD/<TARGET_NAME>/<TOOLCHAIN_NAME>` directory: `<project-name>.bin` and `<project-name>_application.bin`. `<project-name>.bin` is the padded bootloader (used for combining with applications). `<project-name>_application.bin` is the original unpadded bootloader.
 
-When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin`. This is the binary bootloader flashes and then jumps to.
+In this example, the update binary for the bootloader is defined in `main.cpp`  as `mbed-os-example-bootloader-blinky_application.bin` (specified under the macro `UPDATE_FILE`). The bootloader looks for this file in the root of the SD card, flashes it to memory and then jumps to the application.
 
 ## Set up application to use bootloader
 
-The next step is to build an application you can combine with your bootloader to create a loadable image. 
+The next step is to build an application you can combine with your bootloader to create a loadable image. Any application works; however, you may wish to start with the [Blinky example](https://github.com/ARMmbed/mbed-os-example-blinky).
```

Comment:
I was just thinking of the scenario that their application is broken of too large in the first place.  If that is the case, then we cannot say for certain that it will work.